libs/core/facets/facet/facet.component.html

```diff
@@ -2,7 +2,9 @@
     <ng-template [ngTemplateOutlet]="customContentTemplate"></ng-template>
 } @else {
     @if (facetTitle) {
-        <span fd-title [headerSize]="5" class="fd-margin-bottom--sm" [id]="titleId">{{ facetTitle }}</span>
+        <span style="display: block" fd-title [headerSize]="5" class="fd-margin-bottom--sm" [id]="titleId">{{
+            facetTitle
+        }}</span>
```

Comment:
This alone won't fix the problem. The title of a facet needs to be a heading. It needs `role="heading"` and `arial-level="..."`.
Right now this span element is just some text that has no semantic meaning to the user, and it should be a heading.